### DISCUSSION TITLE
Password validation by HTTP validate request

### DISCUSSION THREAD

**Discussion by karlosgm75** [zitadel/zitadel#6918] [Q&A]
Is there any Zitadel feature for validating user credential in legacy system using an HTTP request like this Keycloak plugin: 
[https://github.com/daniel-frak/keycloak-user-migration](https://github.com/daniel-frak/keycloak-user-migration) ?

**Comment by mffap** [comment]
You can use Actions to query your legacy system via a http request and add information to the user: 
https://zitadel.com/docs/concepts/features/actions
